telegram/replykeyboardremove.py

```diff
@@ -28,7 +28,9 @@ class ReplyKeyboardRemove(ReplyMarkup):
     hidden immediately after the user presses a button (see :class:`telegram.ReplyKeyboardMarkup`).
 
     Attributes:
-        remove_keyboard (:obj:`True`): Requests clients to remove the custom keyboard.
+        remove_keyboard (:obj:`True`): Requests clients to remove the custom keyboard. (user will
```

Comment:
maybe make a note from this. Feels cleaner than the way they did in the official docs